### PR TITLE
Adding multi_dot to pytorch frontend

### LINKED ISSUE #11952: multi_dot

### PR DESCRIPTION
**Pull request by abdulasiraj**
closes #11952

### CODE REVIEW

**Comment by abdulasiraj** [comment]
Hi @ZoeCD,
This PR is ready for review. Can you please have sometime?

**Comment by abdulasiraj** [comment]
Hi @ZoeCD,
Sorry for disturbing again. Couldn't you find sometime?

File: ivy_tests/test_ivy/test_frontends/test_torch/test_linalg.py [hunk 1]
```diff
@@ -1094,3 +1094,68 @@ def test_torch_matmul(
...
+        )
+    )
+
+    return input_dtype, [matrix_1[1][0], matrix_2[1][0], matrix_3[1][0]]
+
+
+@handle_frontend_test(
+    fn_tree="torch.linalg.multi_dot",
+    dtype_x=_generate_multi_dot_dtype_and_arrays(),
+    # test_gradients=st.just(False),
```
Comment (ZoeCD) [inline ivy_tests/test_ivy/test_frontends/test_torch/test_linalg.py:1142]: Could you please remove the comment?

File: ivy_tests/test_ivy/test_frontends/test_torch/test_linalg.py [hunk 2]
```diff
@@ -1094,3 +1094,68 @@ def test_torch_matmul(
...
+)
+def test_multi_dot(
```
Comment (ZoeCD) [inline ivy_tests/test_ivy/test_frontends/test_torch/test_linalg.py:1144]: Also, could you please change the test signature to follow the style of the others? `test_torch_multi_dot`

**Review by ZoeCD — CHANGES_REQUESTED** [review]
There are some conflicting files, so please update your branch to resolve them. Other than those minor details, looks good to me!

**Review by ZoeCD — CHANGES_REQUESTED** [review]
You forgot to delete the commented line. And the test is failing now, I guess hypothesis found an edge case or something. This is the failure message `AssertionError:  the results from backend tensorflow and ground truth framework torch do not match`
Could you please check that out? You can see more information in the details section of the failing intelligent tests

**Comment by abdulasiraj** [comment]
I have checked it. The problem is with underlying function defined in back end API. Because I'm just wrapping the function.

**Pull request merged**